composer.json

```diff
@@ -16,6 +16,9 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3"
     },
+    "conflict": {
+        "silverstripe/auditor": "<2.4.0"
```

Comment:
Why this conflict? Can you link to a failing CI run or something that caused you to add this?

Reply:
PHPUnit 9 conflict on prefer-lowest build
`implode(): Passing glue string after array is deprecated. Swap the parameters`
https://github.com/creative-commoners/cwp-pdfexport/runs/7330242616?check_suite_focus=true#step:12:68

Reply:
That is fixed in 2.2.1

Reply:
Updated